source/containers.tex

```diff
@@ -11955,13 +11955,13 @@
 
     template <class Alloc>
       flat_set(flat_set&& m, const Alloc& a)
-        : c{container_type(std::move(m.c), a)}
-        , compare{std::move(m.compare)}
+        : c(std::move(m.c), a)
```

Comment:
Note: this should actually use _uses-allocator_ construction, not just tack the `a` on the end. But that'll be a different pull request.
https://wg21.link/p0591r4 plus guaranteed copy elision will make that diff pretty painless.